android/src/main/java/com/appfolio/extensions/ContextExtensions.kt

```diff
@@ -43,7 +43,7 @@ fun Context.getUploadTask(
 
         UploadServiceLogger.debug(
             component = UploadManager.TAG,
-            uploadId = "",
+            uploadId = creationParameters.params.id,
```

Comment:
this is just debug logging - not sure why upload id was previously left out of this logging but want to include it with this PR